recipe/meta.yaml

```diff
@@ -34,20 +41,31 @@ requirements:
 
 test:
   commands:
-    - test -f ${PREFIX}/lib/libtiff.a  # [not win]
-    - test -f ${PREFIX}/lib/libtiffxx.a  # [not win]
-    - test -f ${PREFIX}/lib/libtiff${SHLIB_EXT}  # [not win]
-    - test -f ${PREFIX}/lib/libtiffxx${SHLIB_EXT}  # [not win]
+    - test -f ${PREFIX}/lib/libtiff.a                            # [unix]
+    - test -f ${PREFIX}/lib/libtiffxx.a                          # [unix]
+    - test -f ${PREFIX}/lib/libtiff.so                           # [linux]
+    - test -f ${PREFIX}/lib/libtiffxx.so                         # [linux]
+    - test -f ${PREFIX}/lib/libtiff.dylib                        # [osx]
+    - test -f ${PREFIX}/lib/libtiffxx.dylib                      # [osx]
```

Comment:
The `SHLIB_EXT` above helps consolidate these a bit across macOS and Linux. Would be nice to keep if that seems reasonable.

Reply:
Yup.